java/.ort.yml

```
@@ -11,10 +11,13 @@ excludes:
   scopes:
     - pattern: "test.*"
       reason: "TEST_DEPENDENCY_OF"
-      comment: Packages for testing only. Not part of released artifacts."
+      comment: Packages for testing only. Not part of released artifacts.
     - pattern: "(spotbugs.*|spotbugsSlf4j.*)"
       reason: "TEST_DEPENDENCY_OF"
-      comment: Packages for static analysis only. Not part of released artifacts."
+      comment: Packages for static analysis only. Not part of released artifacts.
     - pattern: "jacoco.*"
       reason: "TEST_DEPENDENCY_OF"
-      comment: Packages for code coverage verification only. Not part of released artifacts."
+      comment: Packages for code coverage verification only. Not part of released artifacts.
+    - pattern: "compileClasspath.*"
+      reason: "TEST_DEPENDENCY_OF"
+      comment: Packages for Gradle compiler only. Not part of released artifacts.
```

```suggestion
      comment: Packages for Gradle only. Not part of released artifacts.
```
or
```suggestion
      comment: Packages for Gradle scripts only. Not part of released artifacts.
```

`gradle` isn't a compiler, it is a build tool like `make` or `cargo`.

---

Agreed, like the first suggestion.